### ISSUE TITLE
User not recognized as member of administrators group

### ISSUE THREAD

**Issue by gwijnja** [MicrosoftDocs/windowsserverdocs#6713]
I'm running Windows 10 Home 21H2, build 19044.1949, with Powershell 5.1.

So I'm _not_ running Windows server, but the [OpenSSH install guide](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse) says:

> Applies to: Windows Server 2022, Windows Server 2019, **Windows 10 (build 1809 and later)**

I'm in the [Prerequisites check](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse?tabs=gui#prerequisites-check) part of the guide. After verifying that PowerShell 5.1 is installed (step 2), the next step says:

> Run the command below. The output will show True when you're a member of the built-in Administrators group.

```PowerShell
(New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent())).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)
```

So I create a local windows account (selecting "I don't have this person's sign-in information", and then "Add a user without a Microsoft account."), and change the account type to Administrator. I log in as that user, open a powershell, and copy and paste the PS-command above.

Result: False.

What am I missing here? The account is definately an administrator. Maybe the command only works with Windows server, and a separate command should be added for Windows 10?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 18a5a4e5-2604-23c2-1374-68a6974170bc
* Version Independent ID: f93809cc-eb5e-6b9b-ed58-c8811a8ee21f
* Content: [Get started with OpenSSH for Windows](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse?tabs=gui)
* Content Source: [WindowsServerDocs/administration/OpenSSH/OpenSSH_Install_FirstUse.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/main/WindowsServerDocs/administration/OpenSSH/OpenSSH_Install_FirstUse.md)
* Product: **windows-server**
* Technology: **administration**
* GitHub Login: @robinharwood
* Microsoft Alias: **roharwoo**

**Comment by Codingrenzie** [comment]
You have to open PS as an administartor and then run the same command

**Comment by gwijnja** [comment]
> You have to open PS as an administartor and then run the same command

But I said I changed the account type to "Administrator", am I not an administrator then?

**Comment by FriedLeg** [comment]
hey I'm dealing with the same problem currently. did you end up figuring it out?

**Comment by gwijnja** [comment]
> hey I'm dealing with the same problem currently. did you end up figuring it out?

No, I have never resolved it. Back then, I was preparing and testing settings a configuration which would later be rolled out on a Windows Server system, and there it worked fine. But I never got it working on Windows 10 Home, so as far as I'm concerned this is still an open issue.

**Comment by Ycq1111** [comment]
You can just run the teminal as administartor.

**Comment by gwijnja** [comment]
> You can just run the teminal as administartor.

Shouldn't be necessary. The [guide](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse) says:

> Run the command below. The output shows True **when you're a member of the built-in Administrators group**.

**Comment by SueMM00** [comment]
I ran the terminal as administrator and it worked, thank you @Ycq1111

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
